kpi/views/v2/data.py

```diff
@@ -215,8 +215,10 @@ def _get_deployment(self):
         Returns the deployment for the asset specified by the request
         """
         if not self.asset.has_deployment:
-            raise serializers.ValidationError(
-                _('The specified asset has not been deployed'))
+            raise serializers.ValidationError({
+                'deployment': _('The specified asset has not been deployed')
```

Comment:
maybe `ValidationError` isn't the best choice? a 400 should be returned if a user tries to access data for a form that hasn't been deployed (i think? another possibility would be to return an empty list).

however, since the user did not POST or PATCH anything, i don't think it makes sense to have this dictionary with field names.